lib/usecases/login_use_case.dart

```diff
@@ -3,6 +3,8 @@ import 'package:lydiaryanfluttersurvey/api/repository/auth_repository.dart';
 import 'package:lydiaryanfluttersurvey/model/response/login_response.dart';
 import 'package:lydiaryanfluttersurvey/usecases/base/base_use_case.dart';
 
+import '../database/shared_preferences_utils.dart';
```

Comment:
Shall we stick to absolute imports for consistency? 😄 
```suggestion
import 'package:lydiaryanfluttersurvey/database/shared_preferences_utils.dart';
```

Reply:
@ryan-conway Automatically imported by AS. 🙈  Let me try to configure the AS formatting.